demo/ipynb-technote/technote.toml

```diff
@@ -0,0 +1,14 @@
+[technote]
+id = "TEST-000"
+series_id = "TEST"
+canonical_url = "https://test-000.example.org"
+github_url = "https://github.com/SPHEREx/test-000"
+date_created = 2025-01-01
+
+[[technote.authors]]
+name.given = "Jonathan"
+name.family = "Sick"
+orcid = "https://orcid.org/0000-0003-3001-676X"
```

Comment:
Is the `orcid` optional?

Reply:
Yes, most of the metadata is optional. The full schema for the `technote.toml` file is at https://technote.lsst.io/user-guide/technote-toml.html